### PR TITLE
Update Github emoji to new icons

### PR DESCRIPTION
**Pull request by ChrisLovering**
## Description

Github updated their icons, so we should update ours too.

Once merged, we should delete the old emoji form the emoji server to open up room

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

**Comment by ChrisLovering** [comment]
Here is a link to the emojis for anyone who doesn't have access to the emoji server, but wants to confirm the IDs.

https://canary.discord.com/channels/267624335836053506/635950537262759947/852299417323438100

**Review by dawnofmidnight — APPROVED** [review]
Lgtm

File: bot/constants.py [hunk 1]
```diff
@@ -190,12 +190,14 @@ class Emojis:
...
     dice_6 = "<:dice_6:755891607680843838>"
 
-    issue = "<:IssueOpen:629695470327037963>"
-    issue_closed = "<:IssueClosed:629695470570307614>"
-    pull_request = "<:PROpen:629695470175780875>"
-    pull_request_closed = "<:PRClosed:629695470519713818>"
-    pull_request_draft = "<:PRDraft:829755345425399848>"
-    merge = "<:PRMerged:629695470570176522>"
+    # These icons are from Github's repo https://github.com/primer/octicons/
+    issue_opened = "<:IssueOpen:852294683731165184>"
```
Comment (Numerlor) [inline bot/constants.py:194]:
```suggestion
    issue_open = "<:IssueOpen:852294683731165184>"
```
I think open fits a bit more to describe the issue's state, it'll also be consistent with the PR open name

File: bot/constants.py [hunk 2]
```diff
@@ -190,12 +190,14 @@ class Emojis:
...
+    issue_closed = "<:IssueClosed:852294683723694140>"
+    issue_draft = "<:IssueDraft:852294683592884284>"  # Not currently used by Github, but here for future.
+    pull_request_open = "<:PROpen:629695470175780875>"
+    pull_request_closed = "<:PRClosed:852294683194294304>"
+    pull_request_draft = "<:PRDraft:852294683861712916>"
+    pull_request_merge = "<:PRMerged:629695470570176522>"
```
Comment (Numerlor) [inline bot/constants.py:200]:
```suggestion
    pull_request_merged = "<:PRMerged:629695470570176522>"
```

**Review by Numerlor — COMMENTED** [review]
Just a couple of naming nitpicks, looks good otherwise.

File: bot/constants.py (hunk 1 above)
Comment (ChrisLovering) [reply bot/constants.py]: Agreed, added in https://github.com/python-discord/sir-lancebot/pull/774/commits/07a8f19be4f1178cca26330d68c778717d267415

File: bot/constants.py (hunk 2 above)
Comment (ChrisLovering) [reply bot/constants.py]: resolved in https://github.com/python-discord/sir-lancebot/pull/774/commits/07a8f19be4f1178cca26330d68c778717d267415

**Review by Numerlor — APPROVED** [review]

File: bot/constants.py [hunk 3]
```diff
@@ -190,12 +190,14 @@ class Emojis:
...
-    issue_closed = "<:IssueClosed:629695470570307614>"
-    pull_request = "<:PROpen:629695470175780875>"
-    pull_request_closed = "<:PRClosed:629695470519713818>"
-    pull_request_draft = "<:PRDraft:829755345425399848>"
-    merge = "<:PRMerged:629695470570176522>"
+    # These icons are from Github's repo https://github.com/primer/octicons/
+    issue_open = "<:IssueOpen:852294683731165184>"
+    issue_closed = "<:IssueClosed:852294683723694140>"
+    issue_draft = "<:IssueDraft:852294683592884284>"  # Not currently used by Github, but here for future.
+    pull_request_open = "<:PROpen:629695470175780875>"
```
Comment (onerandomusername) [inline bot/constants.py:197]: This emoji, pull_request_open, should be reuploaded.
Current one is below:
image

New one:
image


However, it doesn't seems like this is updated on the octicon repo yet

File: bot/constants.py [hunk 4]
```diff
@@ -190,12 +190,14 @@ class Emojis:
...
+    pull_request_closed = "<:PRClosed:852294683194294304>"
+    pull_request_draft = "<:PRDraft:852294683861712916>"
+    pull_request_merged = "<:PRMerged:629695470570176522>"
```
Comment (onerandomusername) [inline bot/constants.py:200]: Same goes for this one.
Current:
image

New:
image

Again, I could not find it in the octicon repo.

**Review by onerandomusername — CHANGES_REQUESTED** [review]
Looks good, just update the last two emojis.

**Comment by Kronifer** [comment]
Hey everyone, regarding the octicon repo I can find all of these, but none are in color

**Comment by Kronifer** [comment]
lmfao forget i said anything, read up in #dev-branding

**Comment by ChrisLovering** [comment]
Here's an updated image with the new icons
image

**Review by Xithrius — APPROVED** [review]

**Pull request merged**